Berksfile

```diff
@@ -2,6 +2,7 @@ source 'https://supermarket.chef.io'
 
 solver :ruby, :required
 
+cookbook 'cron', git: 'git@github.com:sous-chefs/cron', branch: 'main'
```

Comment:
As I mentioned on the other PR, please remove this and just let Berkshelf pull this from the supermaket.

Reply:
Removed cron from berks